### ISSUE TITLE
Not an issue, but I am banned on TShock and GitHub has no private messaging.

### ISSUE THREAD

**Issue by TeeriaLLC** [Grandpa-G/RemoteAdminConsole#3]
I was wondering if you could create a plugin that can limit maximum mobs on a TShock server because the current TShock lags like a big sheep when there's 200 monsters, hence it should be limited (possibly set by a config) and if you possibly could create a plugin which disables death messages & join/leave messages.

If you need money for this, i'd gladly donate.

**Comment by Grandpa-G** [comment]
Would you mind telling me why you got banned from TShock?

**Comment by TeeriaLLC** [comment]
I got temporarily banned from TShock Forums and Slack by a butthurt admin who couldn't take a joke on Slack regarding the Kappa emote & after I got unbanned I got instantly banned again so I got pissed and retrieved back my donations I made towards TShock due to their shitty behavior and thus got permanently banned for "stealing" money.

**Comment by TeeriaLLC** [comment]
A plugin based on this wouldn't actually make sense as of Terraria/TShock even is right now. TShock is broken but they are just being stubborn about the fact that it is. Every server has only around 20~30 people online also simply because it starts lagging very badly around these numbers.

**Comment by Grandpa-G** [comment]
I don’t think it can be done from the TShock side. It is a limit set in Terraria game and TShock plugin can’t get to that. Maybe a client mod might do something. Sorry.

**Comment by TeeriaLLC** [comment]
Then please do tell Wolfje or the people at TShock to fix this cancerous Packet.
https://gyazo.com/a0d104437e7c1bec54903389dfbaaa10

https://tshock.atlassian.net/wiki/display/TSHOCKPLUGINS/Packet+Documentation#PacketDocumentation-UpdateTileEntity[86]

Because ironically this packet causes every server to lag except for Panther's server.

**Comment by Grandpa-G** [comment]
Sorry, but since I really don’t know the issues here, please contact Wolfje directly.

**Comment by TeeriaLLC** [comment]
Well to get back on that, I done what I could and disabled packet 86 which did get rid of some lag. I do have a request to make on a plugin (whether it costs money yes or no) if you could possibly create something like ClearInvSSC that actually doesn't cause lag because that plugin was made by Icy and is outdated and seems to cause a lot of lag.
